frontend/src/app/components/work-packages/wp-breadcrumb/wp-breadcrumb-parent.html

```diff
@@ -6,19 +6,25 @@
     class="wp-breadcrumb-parent breadcrumb-project-title nocut">
     <span [textContent]="parent.name"></span>
   </a>
-  <accessible-by-keyboard
-    (execute)="open()"
-    *ngIf="canModifyParent()"
-    [linkTitle]="parent ? text.edit_parent : text.set_parent"
-    linkClass="wp-relation--parent-change -no-decoration hide-when-print icon-small {{ parent ? 'icon-edit icon5' : 'icon-add icon4' }}">
+  <button
+      *ngIf="canModifyParent()"
+      role="button"
```

Comment:
I think `role="button"` is not needed, since we're already dealing with a button. Important is `type="button"`, so that it doesn't become a `type="submit"` by default and submit some form it happens to be inside of

Reply:
Yeah I messed up the search/replace, I wanted to make it a `type=button` in the first place :sweat_smile: